DSCResources/MSFT_xNetworkTeam/MSFT_xNetworkTeam.psm1

```diff
@@ -0,0 +1,230 @@
+﻿if (Test-Path "${PSScriptRoot}\${PSUICulture}")
+{
+    Import-LocalizedData -BindingVariable LocalizedData -filename MSFT_xNetworkTeam.psd1 -BaseDirectory "${PSScriptRoot}\${PSUICulture}"
+}
```

Comment:
:) that is precisely the reason I had a dump of all scripts at the beginning of the PSM1. So, removing that completely got me into this bug. I will add the conditional load of en-US.